middleware/lib/middleware/auth.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const router = require('express').Router();
+const jwt = require('express-jwt');
+const jwksRsa = require('jwks-rsa');
+
+const AUTH0_BASE = 'https://total-loss-process.auth0.com';
+
+/**
+ * Auth0 JWT authentication middleware.
+ */
+const checkJwt = jwt({
+  secret: jwksRsa.expressJwtSecret({
+    cache: true,
+    rateLimit: true,
+    jwksUri: `${AUTH0_BASE}/.well-known/jwks.json`,
+  }),
+  audience: `${AUTH0_BASE}/api/v2/`,
+  issuer: AUTH0_BASE,
```

Comment:
@dbartalino Was the trailing slash important here? If so, we'll need to add that back in.

Reply:
You have the backslashes in each place you replaced the URL with the variable, so this should be fine.

Reply:
`issuer:` is missing a backslash where it had one before.